first/utils/docker_runner.py

```diff
@@ -0,0 +1,16 @@
+"""Utility module for interacting with the docker daemon"""
+
+import subprocess
+
+def run_docker_container(repo, branch, num_workers, img_url):
```

Comment:
Add documentation to this function as well as types on both parameters and return value